README.md

```diff
@@ -30,14 +27,25 @@ There are many reasons to use Seyfert, but they dont all fit in this tiny readme
 
 ## Installation
 > [!NOTE]
-> You **NEED** Node.js 18>= for this to work, Node.js 20 LTS and Bun latest are recommended
+> Node v18>= is required (or v16 with some flags), Bun/Node LTS recommended.
```

Comment:
if you say "some flags" you should mention which flags

Reply:
well